docs/annotations.md

```diff
@@ -0,0 +1,17 @@
+Annotations
+===========
+
+Unless otherwise noted, these annotations can be applied to any supported source type
+(e.g. service, ingress, etc.). For the full list, see the [CLI Usage](cli-usage.md) documentation.
```

Comment:
Sorry ... found one more thing. Please remove the reference to `cli-usage.md`.

Reply:
@dacohen please take a look when you have some time. Thanks!

Reply:
Fixed